### ISSUE TITLE
Allow to create script in UI instead of having blank scripts

### ISSUE THREAD

**Issue by paulmcauley** [Androguide/Universal-init.d#4]
I don't like the way you have left by default 3 blank files called "no",  "scripts"  and  "found". This is messy and it is an extra step to delete these. 

I think it would be better instead to have a "+" or "Create a new script"  button in the "Manage your init.d scripts" section to allow you to easily create and name a new script. After all,  you have already created a UI to edit scripts so it should be trivial to add an option to create them too! A rename function is also needed.

**Comment by yurikleb** [comment]
I agree with paul4ig being able to add / rename scripts would be great!
